addons/mail/static/src/models/composer/composer.js

```diff
@@ -376,6 +376,11 @@ function factory(dependencies) {
             if (!this.thread) {
                 return;
             }
+            const suggestionDelimiters = ['@', ':', '#', '/'];
+            const lastInputChar = this.textInputContent.substring(this.textInputCursorStart - 1, this.textInputCursorStart);
+            if (suggestionDelimiters.indexOf(this.suggestionDelimiter || lastInputChar) !== -1) {
+                return;
+            }
```

Comment:
The proper condition to detect a command is `_getCommandFromText`. (`/` must be first char and first word must be part of a command name).